### PR TITLE
DogoNews: Switch to new media template group, implement new design

### PR DESCRIPTION
**Pull request by dogomedia**
This is the first pass at fixing #1867.

### CODE REVIEW

**Comment by dogomedia** [comment]
The current footer matches the existing news tile's footer.  Not sure how to add the icons to match the screenshot @moollaza added to #1867 

screen shot 2015-06-19 at 10 39 27 pm

**Comment by moollaza** [comment]
@dogomedia this is looking awesome!

>Not sure how to add the icons to match the screenshot

Sorry about that, docs are underway for icon usage. In the interim I can just tell you what to do :)

Please use `<span class="ddgsi ddgsi-clock"></span>` for the clock and `<span class="ddgsi ddgsi-user"></span>` for the person icon.

You can also use the classes `pull-left` and `pull-right` to float the author to the left and the time to the right.

File: share/spice/dogo_news/dogo_news.js [hunk 1]
```diff
@@ -28,12 +28,16 @@
...
                         image: thumb,
                         relative_time: moment(item.published_at).fromNow()
                     };
                 },
                 templates:{
-                    item: 'news_item'
+                    group: 'media',
+                    options: {
+                        footer: Spice.dogo_news.footer
+                    }
```
Comment (moollaza) [inline share/spice/dogo_news/dogo_news.js]: Please use a `variants` object to increase the site of the tile snippet:

```js
options: {
  ...
},
variants: {
    tileSnippet: "large"
}
```

File: share/spice/dogo_news/footer.handlebars [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<div class="dogo_news_tile_footer">
```
Comment (moollaza) [inline share/spice/dogo_news/footer.handlebars]: We actually wrap the footer in a div with class `tile__foot`. We can probably remove this wrapping div and adjust the CSS above

**Comment by joe1chen** [comment]
@moollaza Updated footer with icons and cleaned up the css.  Also added ```tileSnippet: "large"```.  This now looks much closer to the screenshot you added.

screen shot 2015-06-22 at 3 14 07 pm

One more remaining question is how you generally prefer to handle the ellipsis for text that is too long?  We used ```source: Handlebars.helpers.ellipsis(item.author, 14)``` to truncate the source to 14 characters.  Is this how you generally prefer to do ellipsis?

**Comment by dogomedia** [comment]
@moollaza Updated footer with icons and cleaned up the css.  Also added ```tileSnippet: "large"```.  This now looks much closer to the screenshot you added.

screen shot 2015-06-22 at 3 14 07 pm

One more remaining question is how you generally prefer to handle the ellipsis for text that is too long?  We used ```source: Handlebars.helpers.ellipsis(item.author, 14)``` to truncate the source to 14 characters.  Is this how you generally prefer to do ellipsis?

**Comment by moollaza** [comment]
>Is this how you generally prefer to do ellipsis?

I hadn't considered those being too long. That should be fine. Alternatively, you could go the CSS route because this is a single line of text, but the JS route is simple and definitely compatible for IE 8+

BTW to increase the size of the snippet further, you can set `meta: { snippetChars: 200 }` which should round out the snippet at about 4 lines.

**Comment by moollaza** [comment]
@abeyang this one is ready for a design review :+1:  It's now using the media template!

**Comment by dogomedia** [comment]
@moollaza Thanks for the tips. We've increased the snippet to 4 lines as you suggested.

screen shot 2015-06-22 at 4 21 12 pm

**Comment by moollaza** [comment]
@dogomedia whoops, that looks like 5 lines! Let's trim that down so we have some space between the footer and snippet :+1:

**Comment by dogomedia** [comment]
@moollaza Good catch.  Dropping it to around 110 chars seems to work best to keep it to 4 lines in both the standard view as well as the grid view.

**Comment by moollaza** [comment]
@dogomedia thanks! Once @abeyang gives us the OK this should be good to merge :+1:

**Comment by abeyang** [comment]
@dogomedia looking good! I'll go more into the visual styles in a bit, but first--

1. How come Meera Dolasia's name is truncated so early? Can we just try to do **first name / last initial**, because otherwise, almost all the authors' names would be truncated...
2. We shouldn't use the detail pane for this; clicking on the tile should just jump straight to the page (like before).

More comments coming soon!

**Comment by dogomedia** [comment]
@abeyang 
1. The name was truncated so early at 14 chars because in the grid view the name would push the time into a second row.  But your suggestion to use first name /last initial is a good one.
2. You are right, the detail pane should not be used.

We will fix both issues immediately.

**Comment by abeyang** [comment]
@dogomedia looking better! Now, for the visuals, let's do this:

1. Let's lose the icons for now, and see how they'll look
2. For the footer (`tile__foot`), let's add this class to it as well: `tx-clr--grey-light`. Should make the footer text become lighter.
3. And for the css, let's change this:

```css
.zci--dogo_news .tile--dogo_news .tile__foot {
  // remove the existing padding-top.. don't think it's doing anything
  // add this instead, to move the footer up a bit:
  padding-bottom: 0.2em;
}
```

Alrighty, hope that all makes sense!

**Comment by dogomedia** [comment]
@abeyang We've removed the icons now.  Regarding 2) we were not sure how to add a custom class to the footer since the ```Spice.dogo_news.footer``` footer we specify in our template options is the content of the ```tile_foot```.  So instead we just added the same grey color to our custom css (```color: #aaa;```) to achieve the same effect.  

screen shot 2015-06-26 at 11 44 18 am

**Comment by moollaza** [comment]
@dogomedia we actually have a new mechanism that let's you apply classes to certain elements:

```js
templates: {
    variants: {
        tileFoot: "tx-clr--grey-light"
    }
}
```

The benefit of our special colour classes is that they change colours based on the chose DDG UI theme, so that way we don't need to specify other colours for the dark theme, or others.

**Comment by dogomedia** [comment]
@moollaza We're now using the ```elClass``` ```tileFoot``` to set the footer class.

**Comment by moollaza** [comment]
@dogomedia excellent. Thanks :+1:

**Comment by abeyang** [comment]
Looks great! :+1:

**Pull request merged**